src/frontend/js/pages/DashboardCertificates/index.tsx

```diff
@@ -1,74 +1,50 @@
+    defaultMessage: 'Certificates',
   },
-  empty: {
-    defaultMessage: 'You have no certificates yet.',
-    description: 'Message displayed when there are no certificates',
-    id: 'components.DashboardCertificates.empty',
+  enrollmentCertificateTabLabel: {
+    id: 'components.DashboardCertificates.enrollmentCertificateTabLabel',
+    description: 'The label of the enrollment certificate tab',
+    defaultMessage: 'Attestation of achievement',
   },
 });
 
-export const DashboardCertificates = () => {
-  const intl = useIntl();
-  const pagination = usePagination({});
-  const certificates = useCertificates(
-    {
-      page: pagination.currentPage,
-      page_size: pagination.itemsPerPage,
-    },
-    { placeholderData: keepPreviousData },
-  );
-  useEffect(() => {
-    if (certificates.meta?.pagination?.count) {
-      pagination.setItemsCount(certificates.meta.pagination.count);
-    }
-  }, [certificates.meta?.pagination?.count]);
-
-  if (certificates.states.error) {
-    return <Banner message={certificates.states.error} type={BannerType.ERROR} />;
-  }
+interface DashboardCertificatesProps {
+  certificateType: CertificateType;
+}
 
+export const DashboardCertificates = ({ certificateType }: DashboardCertificatesProps) => {
   return (
     <div className="dashboard-certificates">
-      {certificates.items.length === 0 && certificates.states.fetching ? (
-        <Spinner aria-labelledby="loading-certificates-data">
-          <span id="loading-certificates-data">
-            <FormattedMessage {...messages.loading} />
-          </span>
-        </Spinner>
-      ) : (
-        <div>
-          {certificates.items.length === 0 ? (
-            <Banner message={intl.formatMessage(messages.empty)} type={BannerType.INFO} />
-          ) : (
-            <>
-              <div
-                className={[
-                  'dashboard-certificates__list',
-                  certificates.states.fetching ? 'dashboard__list--loading' : '',
-                ].join(' ')}
-              >
-                {certificates.items.map((certificate) => (
-                  <DashboardItemCertificate key={certificate.id} certificate={certificate} />
-                ))}
-              </div>
-              <Pagination {...pagination} />
-            </>
-          )}
-        </div>
-      )}
+      <Tabs.Header>
+        <Tabs.Tab
+          name="order-certificate-tab"
+          initialIsActive={certificateType === CertificateType.ORDER}
+          href={LearnerDashboardPaths.ORDER_CERTIFICATES}
+        >
+          <FormattedMessage {...messages.orderCertificateTabLabel} />
+        </Tabs.Tab>
+        <Tabs.Tab
+          name="enrollment-certificate-tab"
+          initialIsActive={certificateType === CertificateType.ENROLLMENT}
+          href={generatePath(LearnerDashboardPaths.ENROLLMENT_CERTIFICATES)}
+        >
+          <FormattedMessage {...messages.enrollmentCertificateTabLabel} />
+        </Tabs.Tab>
```

Comment:
I just remember that we talked to hide this tab if the user has no certificate of this kind. Maybe we could do that in another PR ?

Reply:
i would prefere yes, it's a different journey.